### PR TITLE
Updated boat description

### PR DESCRIPTION
**Pull request by Iamgoofball**
Added a warning for.writers

### CODE REVIEW

**Comment by TehZombehz** [comment]
A warning that came too late.

**Review by Cyberboss — APPROVED** [review]

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -348,10 +348,9 @@
 
 
 //Boat
-
 /obj/vehicle/lavaboat
 	name = "lava boat"
-	desc = "A boat used for traversing lava."
+	desc = "A boat used for traversing lava. Take care to not spend too many years on the boat."
```
Comment (nullbear) [inline code/modules/mining/lavaland/necropolis_chests.dm]: as opposed to dog years???

**Comment by Militaires** [comment]
~~delet this~~

is gud quickmerge

**Comment by Cheridan** [comment]
>Goof can't even finish a PR that's just 1 sentence

**Comment by Iamgoofball** [comment]
it's ready though

i choose not to change that comment

File: code/modules/mining/lavaland/necropolis_chests.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/mining/lavaland/necropolis_chests.dm]: nah

**Comment by phil235** [comment]
Come on, do what Kor said.

**Comment by Iamgoofball** [comment]
Then the joke is even less obvious

**Comment by Iamgoofball** [comment]
Never getting off the boat

**Pull request closed without merging**